pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -80,6 +80,7 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   READINESS_CHECK_OK_CALLS("readinessCheck", true),
   READINESS_CHECK_BAD_CALLS("readinessCheck", true),
   QUERIES_PREEMPTED("query", true),
```

Comment:
Since it hasn't been that much time this has existed in code, I suggest we change the naming to `QUERIES_KILLED`. 

I anticipate us using the terminology of `"PREEMPTED / PREEMPTION"` in the workload management work where we preempt the resource / query etc to better manage resources between queries

Reply:
fixed